manifests/registry-config.yaml

```diff
@@ -16,3 +16,4 @@ spec:
       volumeSource:
         emptyDir: {}
   replicas: 1
+  rolloutStrategy: RollingUpdate
```

Comment:
We don't use this file and I'm going to remove it. It's better to keep this file as is to avoid possible conflicts between PRs.

Reply:
done